api/jobs.yaml

```diff
@@ -165,6 +167,11 @@ definitions:
         description: Fields on ExtendedQueryFields which are queryable
         items:
           type: string
+  DisplayField:
+    description: Map of field name to display name
```

Comment:
ultranit: I'd say description of a display field; could extend this with more later potentially

Reply:
Done.